kroxylicious-app/pom.xml

```diff
@@ -138,6 +138,13 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Would this be a candidate for pushing up into the parent's plugin management?  I think we'd always want UTF-8 in all our artefacts rather than platform encoding.

Reply:
Yep, I think we'd want that by default anywhere we filter resources. Moved up.